content/applications/inventory_and_mrp/manufacturing/workflows/continuous_improvement.rst

```diff
@@ -0,0 +1,233 @@
+app. Creating a survey, and sending it to customers who receive an updated product, increases the
+likelihood of receiving relevant feedback about the product.
+
+.. admonition:: Example workflow: coat rack product improvement
+   :class: alert alert-success
+
+   *Wood Hut* is a manufacturer of fine wood products. They are committed to manufacturing products
+   of the highest-possible quality, and are always looking for ways to improve the products they
+   sell, along with the processes used to create them.
+
+   Wood Hut uses the Odoo platform to manage every element of their production, fulfillment, and
+   customer satisfaction processes. They have developed a custom product improvement workflow that
+   incorporates the Helpdesk, Quality, PLM, and Manufacturing apps.
+
+   One of Wood Hut's most popular products is their *coat rack*. It's made entirely of oak, and
+   customers describe it as "sleek and elegant." However, recent customer feedback about the coat
+   rack has brought attention to quality issues that necessitate revising the current manufacturing
+   process.
+
+   The product revision workflow begins when the customer service team receives a ticket in the
+   Helpdesk app from a customer having problems with the coat rack she purchased. The customer,
+   Abigail Peterson, has found that her coat rack falls over when more than five coats are hanging
+   from it. This is a major issue, as the coat rack has enough dowels for six coats.
+
+   .. image:: continuous_improvement/helpdesk-ticket.png
+      :align: center
+      :alt: A Helpdesk ticket about an issue with the coat rack product.
+
+   Marc, the customer service employee assigned to the helpdesk ticket, opens the Quality app, and
+   creates a new quality alert. He tags the *Production Quality Team* and assigns Julie Andreson as
+   the quality employee responsible for the alert.
+
+   Julie reviews the alert, and consults with her team about the best course of action. They decide
+   that it is necessary to revise the product's |BoM| to prevent the issue from occurring in the
+   future, which Julie notes in the :guilabel:`Corrective Actions` tab of the quality alert.
+
+   .. image:: continuous_improvement/quality-alert.png
+      :align: center
+      :alt: A quality alert created about the issue with the coat rack product.
+
+   Then, Julie messages product engineer, Joe Kazan in the chatter of the quality alert to bring it
+   to his attention. Joe opens the |PLM| app and creates a new |ECO|, noting the problem with the
+   coat rack, and suggesting that a change to the product's |BoM| may be necessary.
+
+   .. image:: continuous_improvement/eco.png
+      :align: center
+      :alt: An ECO created to update the coat rack product's BoM.
+
+   Joe clicks *Start Revision*, and then the *Revision* smart button to open version two of the coat
+   rack's |BoM|. This |BoM| was created alongside the |ECO|, and remains archived until it is
+   approved.
```

Comment:
Since this example is walking through the process, I think it is safe to use :guilabel:s here
```suggestion
   Joe clicks :guilabel:`Start Revision`, and then the :guilabel:`Revision` smart button to open
   version two of the coat rack's |BoM|. This |BoM| was created alongside the |ECO|, and remains
   archived until it is approved.
```